### ISSUE TITLE
数据错误

### ISSUE THREAD

**Issue by lyc728** [VDIGPKU/IterNet#3]
<img width="525" alt="企业微信截图_1650527878603" src="https://user-images.githubusercontent.com/88084847/164407789-bee34ba6-0428-4766-a47b-de0c67b51b09.png">
你好，我加载自己的数据报这个错误，我数据是转成lmdb的形式

**Comment by lyc728** [comment]
<img width="188" alt="企业微信截图_16505291682993" src="https://user-images.githubusercontent.com/88084847/164411765-bd17b104-744f-4297-800f-137ab2fc8756.png">
你好，我注意到你这里是训练数字，英文的字符，并没有对汉字进行处理

**Comment by CShuai-USTB** [comment]
eval loss =  2.528,  ccr =  0.595,  cwr =  0.125,  ted = 16603.000,  ned =   5534,  ted/w =  1.216
你好，我用Evaluation的命令测试测试集，为什么得到的这几个参数没有说明是什么？我不明白这几个参数的意义。为什么没有accuracy这个指标？
